src/Atc.Rest.MinimalApi/Extensions/EndpointDefinitionExtensions.cs

```diff
@@ -33,9 +33,46 @@ public static class EndpointDefinitionExtensions
         services.AddSingleton(endpointDefinitions as IReadOnlyCollection<IEndpointDefinition>);
     }
 
+    /// <summary>
+    /// Adds the endpoint definitions to the specified service collection by scanning the assemblies of the provided marker types.
+    /// This method looks for types that implement the <see cref="IEndpointDefinition"/> interface and are neither abstract nor an interface,
```

Comment:
and IEndpointAndServiceDefinition interfaces (with cref on IEndpointAndServiceDefinition)